setup.rst

```diff
@@ -107,6 +107,11 @@ Open your browser and navigate to ``http://localhost:8000/``. If everything is
 working, you'll see a welcome page. Later, when you are finished working, stop
 the server by pressing ``Ctrl+C`` from your terminal.
 
+.. tip::
+
+    You can start the local web server in detach mode by adding `-d` option.
```

Comment:
Better use the `--daemon` (more precise, maybe `--daemon (or -d)`)
and use the word background instead of detach, wdyt?

The tool doc is `  --daemon, -d           Run the server in the background`